README.md

```diff
@@ -68,18 +67,31 @@ Note: This is an example of a non-normative note.
 
 # Overview
 
-Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP. Carrying GraphQL in HTTP does not mean that GraphQL overrides existing
-semantics of HTTP but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
+Serving GraphQL over HTTP provides the ability to use the full advantages of GraphQL with the rich feature set of HTTP.
+Carrying GraphQL in HTTP does not mean that GraphQL overrides existing semantics of HTTP,
+but rather that the semantics of GraphQL over HTTP map naturally to HTTP semantics.
 
-GraphQL naturally follows the HTTP request/response message model providing a GraphQL request in an HTTP request and
-GraphQL response in an HTTP response.
+GraphQL naturally follows the HTTP request/response message model,
+providing a GraphQL request in an HTTP request and GraphQL response in an HTTP response.
 
 # URL
 
-A GraphQL server operates on a single URL and all GraphQL requests for a given service should be directed
-at this URL. Other protocols may also use that URL.
+A GraphQL over HTTP compliant server MUST designate at least one URL that handles GraphQL requests.
+
+All GraphQL operations that are available in a schema MUST be available on all URLs that are designated to it.
+That means the client can send all their GraphQL requests to a single endpoint, regardless
+of the contained Operations, Fragments or Variables.
+
+Multiple URLs MAY exist on a server to handle GraphQL requests, potentially serving different
+but independent schemas. It is allowed for the same GraphQL schema to be available on multiple URLs.
+
+Other protocols or services may also use those URLs, as long as they don't conflict with the
+server's responsibility to handle GraphQL requests.
```

Comment:
I believe `protocol` is not the right word here.  GraphQL is not a protocol.  HTTP is a protocol.  How about this:

```suggestion
Those URLs MAY also handle other types of requests, as long as they don't conflict with the
server's responsibility to handle GraphQL requests.
```

Reply:
Well, it means the URL may also be used to handle FTP or other protocols. Maybe this would be even for generic:

```suggestion
Those URLs MAY also be used for other purposes, as long as they don't conflict with the
server's responsibility to handle GraphQL requests.
```

Reply:
This wording is much better.